src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -154,8 +154,10 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('location', get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
         c.argument('account_type', help='The storage account type', arg_type=get_enum_type(t_account_type))
         c.argument('account_name', acct_name_type, options_list=['--name', '-n'], completer=None)
-        c.argument('kind', help='Indicates the type of storage account.',
-                   arg_type=get_enum_type(t_kind, default='storage'))
+        c.argument('kind', help='Indicates the type of storage account.', min_api="2017-10-01",
```

Comment:
Can you share the reason about changing the default to V2?

Reply:
Most features are only available for storageV2 storage account. In this way, StorageV2 is a general purpose storage account.
In addition, there are requirements from other team to have such default value and it is the same with PSH behavior.